tests/Unit/Enums/AppliesToTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Support\Tests\Unit\Enums;
+
+use Tipoff\Support\Enums\AppliesTo;
+use Tipoff\Support\Tests\TestCase;
+
+class AppliesToTest extends TestCase
+{
+    /** @test */
+    public function can_get_array_of_options()
```

Comment:
This test (and the same test on LayoutType) should really be combined into a single test on `BaseEnum` since that's where the code lives.  There's no code in the actual enums themselves. so really don't need a distinct test case for any of them once the base is covered.

Reply:
@Kenny-MWI Could you make this update in a new PR? I'm going to go ahead and merge this one.